### PR TITLE
Add initial plugin code

### PR DESCRIPTION
**Pull request by bfcoder**

### CODE REVIEW

File: lib/canvas_customize_passwords/engine.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  DESCRIPTION = "Enables custom password requirements".freeze
+
+  class Engine < ::Rails::Engine
+    config.paths["lib"].eager_load!
+
+    config.to_prepare do
+      Canvas::Plugin.register(
+        :canvas_customize_passwords,
+        nil,
+        name: -> { I18n.t(:canvas_fips_name, NAME) },
```
Comment (srphillips) [inline lib/canvas_customize_passwords/engine.rb]: Copy paste error here and the next line with `canvas_fips_name` I think

**Review by srphillips — CHANGES_REQUESTED** [review]

File: lib/canvas_customize_passwords/engine.rb (hunk 1 above)
Comment (bfcoder) [reply lib/canvas_customize_passwords/engine.rb]: dang it.

Comment (bfcoder) [reply lib/canvas_customize_passwords/engine.rb]: fixed

**Review by srphillips — APPROVED** [review]
Looks good.  Lets get it into the build process for testing

**Pull request merged**